modules/aws/vpc/vpc.tf

```diff
@@ -63,6 +63,13 @@ EOF
       "Action": "*",
       "Effect": "Allow",
       "Resource": "arn:${var.arn_region}:s3:::*-g8s-*"
+    },
+    {
+      "Sid": "Giantswarm-Audit-Rule",
+      "Principal": "*",
+      "Action": "*",
+      "Effect": "Allow",
+      "Resource": "arn:${var.arn_region}:s3:::*-giantswarm-*/*"
```

Comment:
Adds the /* at the end so objects can be updated (important for auditing)

Reply:
Are we sure the bucket name format will always follow this `*-giantswarm-*/*` ?

Reply:
I think it's the aim of the PR. If there is no "-giantswarm-" in the bucket name, access is forbidden.

Reply:
Yes, Vodafone uses account-id-giantswarm-audit-logs and I think it makes sensé that we en force that in the future

Reply:
@stone-z I thing I did not express the issue properly, I;m sorry.
What this changes does is to not block access to s3 buckets containing giantswarm from our management clusters at the VPC level ([s3 gateway endpoint](https://docs.aws.amazon.com/vpc/latest/privatelink/vpc-endpoints-s3.html) that ensure network access goes through the internal amazon network and not the internet) but there are still iam roles that prevents any action on buckets that is not authorized. The role for fluent-bit at customers is limited to s3 calls to this bucket anyway :)

But I agree we can enforce that the buckets should contain giantswarm-audit-logs.

I would like in a near future to have this bucket using crossplane so we provide audit logs by default but that's a far away dream